examples/experimental/snowflake_feedbacks.ipynb

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        "account": "xxx",
</pre><p>"xxx" -&gt; "..."</p><p> </p><p>also change "role" to "..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1452/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>